test/project_types/script/layers/infrastructure/languages/assemblyscript_project_creator_test.rb

```diff
@@ -33,194 +47,21 @@
   describe ".setup_dependencies" do
     subject { project_creator.setup_dependencies }
 
-    it "should write to package.json" do
-      context.expects(:capture2e).returns([JSON.generate("2.0.0"), OpenStruct.new(success?: true)]).times(3)
-      context.expects(:write).with do |_file, contents|
-        payload = JSON.parse(contents)
-        build = payload.dig("scripts", "build")
-        expected = [
-          "shopify-scripts-toolchain-as build --src src/shopify_main.ts",
-          "--binary build/script.wasm --metadata build/metadata.json",
-          "-- --lib node_modules --optimize --use Date=",
-        ].join(" ")
-        expected == build
-      end
-
-      subject
-    end
-
-    it "should fetch the latest extension point version if the package is not versioned" do
-      context
-        .expects(:capture2e)
-        .with("npm --userconfig ./.npmrc config set @shopify:registry https://registry.npmjs.com")
-        .returns(fake_capture2e_response)
-      context
-        .expects(:capture2e)
-        .with("npm --userconfig ./.npmrc config set engine-strict true")
-        .returns(fake_capture2e_response)
-      context
-        .expects(:capture2e)
-        .with("npm show @shopify/extension-point-as-fake version --json")
+    it "should setup dependencies" do
+      Script::Layers::Infrastructure::Languages::ProjectCreator.any_instance
+        .expects(:setup_dependencies)
+        .with
         .once
-        .returns([JSON.generate("2.0.0"), OpenStruct.new(success?: true)])
-
-      sdk = mock
-      sdk.expects(:sdk_version)
-      sdk.expects(:toolchain_version)
-      sdk.expects(:versioned?).once.returns(false)
-      sdk.expects(:package).twice.returns("@shopify/extension-point-as-fake")
-      extension_point.expects(:sdks).times(5).returns(stub(all: [sdk], assemblyscript: sdk))
 
-      subject
-      version = JSON.parse(File.read("package.json")).dig("devDependencies", "@shopify/extension-point-as-fake")
-      assert_equal "^2.0.0", version
-    end
-
-    it "should set the specified package version when the package is versioned" do
       context
         .expects(:capture2e)
-        .with("npm --userconfig ./.npmrc config set @shopify:registry https://registry.npmjs.com")
+        .with(Script::Layers::Infrastructure::Languages::AssemblyScriptProjectCreator::NPM_SET_REGISTRY_COMMAND)
         .returns(fake_capture2e_response)
       context
         .expects(:capture2e)
-        .then.with("npm --userconfig ./.npmrc config set engine-strict true")
+        .with(Script::Layers::Infrastructure::Languages::AssemblyScriptProjectCreator::NPM_SET_ENGINE_STRICT_COMMAND)
```

Comment:
In these cases where we don't do anything with the response, I don't think we don't need to return anything in the test.